test/testdata/templateservicebroker/templateservicebroker-template.yaml

```diff
@@ -0,0 +1,114 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: template-service-broker
+parameters:
+- name: IMAGE
+  value: openshift/origin:latest
+- name: TSB_TEMPLATE_NAMESPACE
```

Comment:
> TSB_TEMPLATE_NAMESPACES plural?

It's a weird thing.  I don't know how to plumb this through with a template to the arg.  I'm going to talk with @sdodson about how we want to handle config long term.  Can I carry it as-is for now knowing it won't survive contact with ansible?